stix_shifter_modules/msatp/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -92,43 +412,92 @@
         "object": "nc_process_file"
       },
       {
-        "key": "process.image_ref",
+        "key": "process.binary_ref",
         "object": "nc_process",
         "references": "nc_process_file"
+      },
+      {
+        "key": "x-oca-event.process_ref",
+        "object": "event",
+        "references": "nc_process"
       }
     ],
     "InitiatingProcessFileName": [
       {
         "key": "file.name",
-        "object": "nc_process_file"
+        "object": "nc_process_file",
+        "transformer": "ToFileName"
+      },
+      {
+        "key": "process.name",
```

Comment:
stix 2.1 doesn't have a name property for process object. it can be mapped like a custom properties `process.x_name`

Reply:
thanks, all process.name references removed